storefront/styles/globals.css

```diff
@@ -180,3 +180,201 @@ img-thumbnail {
   color: #0070f3;
   transform: scale(1.1);
 }
+
+.product_detail_tr {
```

Comment:
I think we should create a separate file to put css for product details page

Reply:
i got it. i had create a separate file for product detail CSS . Please have a check